### PR TITLE
libwmf: update to 0.2.12

### PR DESCRIPTION
**Pull request by ndowens**
Change source as original upstream is dead also adopt.

Tested with abiword with a wmf file and it worked fine

### CODE REVIEW

**Comment by ericonr** [comment]
Have you checked that all patches made it to the new upstream?

**Comment by ndowens** [comment]
Went through comparing and the lines that were added in the patches are in the new version's file, the code is a little different. But it seems so IMO

File: srcpkgs/libwmf/patches/04_gd-gd_clip.c-use-after-free-cve-2009-1364.patch [hunk 1]
```diff
@@ -1,11 +0,0 @@
---- src/extra/gd/gd_clip.c
-+++ src/extra/gd/gd_clip.c
-@@ -70,6 +70,7 @@
- 	{	more = gdRealloc (im->clip->list,(im->clip->max + 8) * sizeof (gdClipRectangle));
- 		if (more == 0) return;
- 		im->clip->max += 8;
-+		im->clip->list = more;
```
Comment (ndowens) [inline srcpkgs/libwmf/patches/04_gd-gd_clip.c-use-after-free-cve-2009-1364.patch:7]: This line was in new version, just the red area wasn't removed

**Comment by biopsin** [comment]
@ndowens Ping to rebase

**Pull request closed without merging**